components/scream/src/interface/scream_scorpio_interface.F90

```diff
@@ -297,6 +297,9 @@ subroutine register_dimension(pio_atm_filename,shortname,longname,length)
     ! Get a new dimension pointer in coord_list
     curr => pio_atm_file%coord_list_top
     do while (associated(curr))
+      if (associated(curr%coord)) then 
+        if(trim(curr%coord%name)==trim(shortname)) call errorHandle("PIO Error: Could not register dimension"//trim(shortname)//" and file: "//trim(pio_atm_filename)//". Already exists.",-999)
```

Comment:
The error msg is slightly unclear: what is already existing, the file or the dimension? Perhaps say

```
"Could not register dimension"//trim(shortname)//" in file "//trim(pio_atm_filename)//". The dimension already exists.")
```

Reply:
fixed